internal/cli/terraform.go

```diff
@@ -264,7 +268,7 @@ func createMainFile(outputDIR string) error {
   required_providers {
     auth0 = {
       source  = "auth0/auth0"
-      version = "1.0.0-beta.1"
+      version = "1.0.0-beta.4"
```

Comment:
Not yet released but anticipating this release. It includes necessary fixes for this feature.